rosdep/python.yaml

```
@@ -412,6 +412,19 @@ pyflakes3:
       packages: [pyflakes]
   rhel: ['python%{python3_pkgversion}-pyflakes']
   ubuntu: [pyflakes3]
+pygeodesy-pip:
+  debian:
+    pip:
+      packages: [pygeodesy]
+  fedora:
+    pip:
+      packages: [pygeodesy]
+  osx:
+    pip:
+      packages: [pygeodesy]
+  ubuntu:
+    pip:
+      packages: [pygeodesy]
```

We can use an [OS wildcard](https://ros.org/reps/rep-0111.html#wildcard-os-name) here to target all platforms which support pip rather than calling them out individually.

```suggestion
  '*':
    pip:
      packages: [pygeodesy]
```